src/readme.graph.md

```diff
@@ -428,6 +428,20 @@ directive:
         let errorDetailsRegex = /(ErrorDetails\s*=\s*)(new.*ErrorDetails\(message\).*)/gmi
         $ = $.replace(errorDetailsRegex, '$1await this.GetErrorDetailsAsync((await response)?.Error, responseMessage)');
 
+        // Prevents null response objects to the output stream for scenarios where response is a model type
+        let responseTypeRegex = /global::System.Threading.Tasks.Task<Microsoft.Graph.PowerShell.Models.\w*> \w*\)[^]*?(WriteObject.*(await response).*;)/gm
+        var responseTypeRegexMatch = $.match(responseTypeRegex);
+        if(responseTypeRegexMatch){
+           responseTypeRegexMatch.forEach((item)=>{
+            var writeObjectRegex = /(WriteObject.*(await response).*;)/gm
```

Comment:
`var writeObjectRegex = /(WriteObject.*(await response).*;)/gm` should be moved outside the foreach loop since it's fixed.